build/lib/helm_require.sh

```diff
@@ -45,15 +45,22 @@ spec:
 REQUIRES_CONFIG_FILE=helm/requires-config.yaml
 SEDFILE=${OUTPUT_DIR}/helm/sedfile
 export IMAGE_TAG
-export HELM_REGISTRY
+export HELM_REGISTRY=$(aws ecr-public describe-registries --region us-east-1  --output text --query 'registries[*].registryUri')
```

Comment:
Is it possible to have two registries? Maybe it's safer to select index 0? It might not be the one we want, but at least the script wouldn't barf.

Also, no check is done here for a failure in the aws command, or a blank return:
```bash
if [ -z "$HELM_REGISTRY" ]; then
    logger --stderr --tag "helm_require.sh" "no helm registry found via lookup"
fi
```

Reply:
At this time ECR public does not support more than one

Reply:
above there are a bunch of bash settings that are *supposed* to fail the entire script if a part fails.